src/porepy/models/material_constants.py

```diff
@@ -580,6 +556,53 @@ def maximum_elastic_fracture_opening(self) -> number:
             self.constants["maximum_elastic_fracture_opening"], "m"
         )
 
+
+class NumericalConstants(MaterialConstants):
```

Comment:
This should not reside inside material_constants, since it does not relate to a material.